src/utils/helpers/date/date.js

```diff
@@ -62,12 +62,12 @@ const DateHelper = {
    * @method formatDateString
    * @param {String} value current value e.g. Wed Aug 23 2017 12:00:00 GMT+0100 (BST)
    * @param {String} formatTo Desired format e.g. YYYY-MM-DD
+   * @param {boolean} utcFormat set whether date should be utc formatted
    * @return {String} formatted date
    */
-  formatDateString: (value, formatTo) => {
-    return (
-      moment(new Date(value).getTime()).format(formatTo)
-    );
+  formatDateString: (value, formatTo, utcFormat = true) => {
```

Comment:
If we have two returns next to each other maybe it should be good to have something like 
```js
if (utcFormat) { 
  return moment(new Date(value).getTime()).format(formatTo)
};

return moment.utc(new Date(value).getTime()).format(formatTo);
```
What do you think about this?

Reply:
Yeah it's more readable in this scenario I suppose, change made